storage/rocksdb/ha_rocksdb.cc

```diff
@@ -372,10 +372,10 @@ static void rocksdb_flush_all_memtables() {
   }
 }
 
-static void rocksdb_delete_column_family_stub(THD *const /* thd */,
-                                              struct SYS_VAR *const /* var */,
-                                              void *const /* var_ptr */,
-                                              const void *const /* save */) {}
+// A stub sysvar update function that does nothing. Used instead of nullptr to
+// indicate that the sysvar is not read-only.
+static void rw_sysvar_update_noop(THD *const, SYS_VAR *const, void *const,
```

Comment:
nit: maybe start with rocksdb_?

Reply:
Done
(I mildly prefer not having prefixes when we have namespaces and in this instance static functions)